sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/SchemaRegistryHelper.java

```diff
@@ -38,29 +41,29 @@ public static void setAccessor(SchemaRegistryModelsAccessor modelsAccessor) {
         accessor = Objects.requireNonNull(modelsAccessor, "'modelsAccessor' cannot be null.");
     }
 
-    public static SchemaProperties getSchemaProperties(SchemasRegisterResponse response) {
-        final SchemasRegisterHeaders headers = response.getDeserializedHeaders();
+    public static SchemaProperties getSchemaPropertiesFromSchemasGetSchemaVersionHeaders(ResponseBase<SchemasGetSchemaVersionHeaders, BinaryData> response) {
```

Comment:
Instead of creating these differently named overloads, can we use the same method name but take headers instead of the whole response?
```java
public static SchemaProperties getSchemaProperties(SchemasGetSchemaVersionHeaders headers) {
}

public static SchemaProperties getSchemaProperties(SchemasQueryIdByContentHeaders headers) {
}
```